core/schema.sql

```diff
@@ -7,6 +7,7 @@
 
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
updating postgres will require re-bundling the windows installer and (i think) mac app.